src/acl.c

```diff
@@ -2254,7 +2254,7 @@ int ACLSaveToFile(const char *filename) {
     /* Create a temp file with the new content. */
     tmpfilename = sdsnew(filename);
     tmpfilename = sdscatfmt(tmpfilename,".tmp-%i-%I",
-        (int)getpid(),(int)mstime());
+        getpid(),mstime());
```

Comment:
the `(int)` casting for `mstime` looks like bug, one that could have no implications on some platforms (calling conventions), and and severe implications on others.

but i think the cat for the `getpid` should remain (in case on some system pid_t isn't an `int`.
let's do that in cluster.c too.

Reply:
oops...I forgot I was developing this feature in Windows...